lib/engines/hckinstall/hckinstall.rb

```diff
@@ -15,23 +15,205 @@ class HCKInstall
 
     attr_reader :platform
 
+    PLATFORMS_JSON = 'lib/engines/hcktest/platforms.json'
+    ISO_JSON = 'lib/engines/hckinstall/iso.json'
+
+    ANSWER_FILES = ['autounattend.xml', 'unattend.xml'].freeze
+    CLIENTS = %w[c1 c2].freeze
+
+    STUDIO_INSTALL_TIMEOUT = 3 * 60 * 60 # 3 hour
+    CLIENT_INSTALL_TIMEOUT = 1 * 60 * 60 # 1 hour
```

Comment:
What's the difference?

Reply:
HLK studio required more time for installation. The longest studio installation time was 2 hours.
The longest client installation time was 30 min.